workflows/spatial.nf

```diff
@@ -112,12 +124,23 @@ workflow SPATIAL {
         .concat( not_bleed_corrected_deconvolution_input )
 
     BAYESTME_DECONVOLUTION( deconvolution_input )
+    ch_versions = ch_versions.mix(BAYESTME_DECONVOLUTION.out.versions)
+
+    SPACEMARKERS( BAYESTME_DECONVOLUTION.out.adata_deconvolved.map { tuple(it[0], it[1]) }.join(data_directory) )
```

Comment:
Does it make sense to move SpaceMarkers after the BTME Spatial Transcriptional Programs if possible?

Reply:
Got it! We can move it in a later change. Approving merge now.